examples/basic/variables.tf

```diff
@@ -1,33 +1,35 @@
-########################################################################################################################
-# Input variables
-########################################################################################################################
+####################
+# Cluster variables
+####################
+variable "cluster_id" {
```

Comment:
Examples needs to be end to end and not have any prereqs. So your example need to actually deploy a cluster instead of taking in an input for an existing cluster. See https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/examples/basic/main.tf as a reference